rundeckapp/grails-spa/packages/ui-trellis/src/app/utilities/locales/en_US.js

```diff
@@ -1039,6 +1039,98 @@ const messages = {
+    "Does not import the project configuration.",
+  "archive.import.importWebhooks.true.title": "Import Webhooks",
+  "archive.import.validateJobRef.true.help":
+    "Throw errors when using a referenced job that does not exist.",
+  "archive.import.importACL.false.help": "Does not import ACL Policies.",
+  "archive.import.validateJobRef.true.title": "Validate Referenced Jobs",
+  "archive.import.importWebhooks.false.title": "Do not import webhooks",
+  "archive.import.importACL.true.help":
+    "Imports any Project ACL Policies, overwriting policies with the same name.",
+  "archive.import.importNodesSource.false.help":
+    "Does not import the Node Sources.",
+  "archive.import.importScm.unauthorized.help":
+    "Not authorized to configure SCM.",
+  "archive.import.importConfig.true.title": "Import Project Configuration",
+  "archive.import.importScm.true.title": "Import Project SCM Configuration",
+  "archive.import.importNodesSource.true.help":
+    "Overwrites the Node Sources with the properties stored in the archive.",
+  "archive.import.importNodesSource.true.title": "Import Node Sources",
+  "archive.import.importNodesSource.failed.message":
+    "Failed to save nodes file, please contact your administrator.",
+  "archive.import.importACL.unauthorized.help":
+    "Not authorized to create ACL Policies.",
+  "archive.import.validateJobRef.false.help":
+    "Import referenced jobs without validating existence.",
+  "archive.import.importScm.true.help":
+    "Import Project SCM Configuration unless the project has SCM configuration.",
+  "archive.import.validateJobRef.label": "Validate Referenced Jobs:",
+  "archive.import.uuidOption.remove.description":
+    "Strip UUIDs from imported jobs.",
+  "archive.import.uuidOption.preserve.description":
+    "Preserve the imported Job UUIDs. Note: If a job with the same UUID exists in another project, the new job will not be imported.",
+  "archive.import.uuidOption.preserve.label": "Preserve UUIDs",
+  "archive.import.uuidOption.remove.label": "Remove UUIDs",
+  "archive.import.uuidOption.label": "Imported Jobs:",
+  // Job Upload Page messages
+  jobUpload: {
+    title: "Upload Job Definition to project",
+    fileSelectLabel: "Select a Jobs definition file.",
+    formatRadioLabel: {
+      xml: "XML format",
+      yaml: "YAML format",
+      json: "JSON format",
+    },
+    dupeOption: {
+      label: "When a Job with the same name already exists:",
+      update: {
+        word: "Update",
+        label: "{0} the existing Job",
+      },
+      skip: {
+        word: "Skip",
+        label: "{0} the uploaded Job",
+      },
+      create: {
+        word: "Create",
+        label: "Always {0} a new Job",
+      },
+    },
+    button: {
+      cancel: "Cancel",
+      upload: "Upload",
+      uploadAnother: "Upload Another Job Definition File...",
+    },
+    uploadingFile: "Uploading File...",
+    jobs: "Job | Jobs",
+    wasWere: {
+      singular: " was",
+      plural: "s were",
```

Comment:
```suggestion
      plural: " were",
```
That extra s was causing all messages to say "jobss"